src/app/header/header.component.html

```diff
@@ -0,0 +1,3 @@
+<p>
+  header works!
```

Comment:
This is not needed .. Please remove it. Thanks..

Reply:
Done